src/trace_service_client.hpp

```diff
@@ -17,10 +17,17 @@ class TraceServiceClient {
     typedef std::function<void (Request, Response, grpc::Status)>
         ResponseCb;
 
-    TraceServiceClient(const std::string& target)
+    TraceServiceClient(const std::string& target, const std::string &trusted_certificate, bool ssl)
     {
-        auto channel = grpc::CreateChannel(
-            target, grpc::InsecureChannelCredentials());
+        auto creds = grpc::InsecureChannelCredentials();
```

Comment:
I don't think we should create such credentials for SSL case at all.

Reply:
Yes, but it is needed for the non-SSL use case, right? or what do you mean?